src/js/components/HeaderBar.tsx

```diff
@@ -13,8 +15,7 @@ export default function HeaderBar() {
       <div className="header-bar-logo-wrapper">
         <div className="header-bar-logo" />
       </div>
-      <span>user-menu component here</span>
-      <span>cluster-menu component here</span>
```

Comment:
you removed my placeholder 😡 

Just kidding, no need for it anymore ;)

Reply:
actually - since you touch this PR anyway, can you please revert this change? :) 
reason is, that it will cause a conflict in daniels PR once this one is merged… (the import also might cause a conflict tho 🤔 )

absolutely **not** blocking